assets/data/hctrl.dat

```diff
@@ -0,0 +1,16 @@
+CHANGE_CYCLING	CONTROL	
```

Comment:
We probably want to gitignore these .dat files once everyone has them, so that we don't end up getting their changes in the repo every time someone tests the controls.